README_OSE.md

```diff
@@ -222,3 +223,6 @@ To override the the defaults, you can set the variables in your inventory:
 ose3-master.example.com openshift_ip=1.1.1.1 openshift_hostname=ose3-master.example.com openshift_public_ip=2.2.2.2 openshift_public_hostname=ose3-master.public.example.com
 ...snip...
 ```
+
+## Parallel image pulls with Docker 1.9+
+If you are using Docker 1.9+, you may consider to disable serial image pulls (or to enable parallel image pulls).
```

Comment:
I wouldn't say both disable serial and enable parallel. Just say enable parallel.

Please say there is an issue with data corruption with parallel pulls using Docker before 1.9. And that starting with 1.9, the corruption issue has been resolved, so it's safe to switch to parallel, but that the default if you do nothing is still serial.